src/Native/FactorizationMachineNative/CMakeLists.txt

```diff
@@ -1,12 +1,16 @@
 project (FactorizationMachineNative)
 
 set(SOURCES
-    FactorizationMachineCore.cpp
+    FactorizationMachineCoreSSE.cpp
```

Comment:
Are we compiling all of source files? Are all of them cross-platform?

Reply:
Yes, we are compiling all the source files. Not all of them are cross-platform. Avx.isSupported (and similar) dispatches the code as needed.